src/components/Main.vue

```diff
@@ -0,0 +1,211 @@
+                                            class="button is-info is-fullwidth submit-button"
+                                            v-on:click="renderIssues" 
+                                            @click = "showSidebar(); hideHeader(); changeWidth()"
+                                        >
+                                            Submit
+                                        </button>
+                                    </div>
+                                </div>
+                                <div v-if="rendered">
+                                    <button
+                                        class="button is-primary is-fullwidth back-button"
+                                        v-on:click="
+                                            () => {
+                                            rendered = false;
+                                            }
+                                        "
+                                    >
+                                        &leftarrow; Again!
+                                    </button>
+                                    <div class = 'blurbs' v-bind:style="{width: widthVal}">
+                                        <Blurb
+                                        v-for="message in messages"
+                                        :key="message.rnd"
+                                        :message="message"
+                                        />
+                                    </div>
+                                </div>
+                            </div>
+                        </div> 
+                        <div class ='sidebar_container' v-if = 'sidebarStatus'>
+                            <div class = 'sidebar'> 
+                                <p> Try to make the following changes: </p> 
+                                <Summary v-for="summary in summaries"
+                                 :key= "summary.rnd"
+                                 :summary ="summary" />
+                            </div> 
+                        </div>
+                    </span> 
+                </div>
+            </div>
+        </div> 
+    </div>
+    
+</template>
+<script>
+import Blurb from "./Blurb";
+import Summary from "./Summary.vue"
+const URL = "http://localhost:5000";
+export default {
+    name: "Main",
+    components: {
+        Blurb,
+        Summary
+    },
+    data() {
+        return {
+            text: "",
+            inputText: "",
+            messages: [],
+            summaries: [],
+            rendered: false,
+            sidebarStatus: false,
+            widthVal: '700px'
+
+        };
+    },
+    methods: {
+        showSidebar() {
+            this.sidebarStatus = true; 
+        },
+        hideHeader() {
+            this.$emit('hideHeader'); 
+        },
+        changeWidth(){
+            if (this.widthVal === '700px'){
+                this.widthVal ='425px';
+            }
```

Comment:
Yeah, this is kind of a kludge. I wonder if there's a more elegant approach. Maybe percentages instead of pixels?